lib/smart_proxy_openscap/openscap_lib.rb

```diff
@@ -13,10 +13,21 @@
 require 'json'
 require 'proxy/error'
 require 'proxy/request'
+require 'smart_proxy_openscap/openscap_exception'
 
 module Proxy::OpenSCAP
   extend ::Proxy::Log
 
+  def self.get_policy_content(policy_id)
+    policy_store_dir = File.join(Proxy::OpenSCAP::Plugin.settings.contentdir, policy_id.to_s)
+    policy_scap_file = File.join(policy_store_dir, "#{policy_id}_scap_content.xml")
+    FileUtils.mkdir_p(policy_store_dir) # will fail silently if exists
+
+    scap_file = policy_content_file(policy_scap_file)
+    scap_file ||= save_or_serve_scap_file(policy_id, policy_scap_file)
```

Comment:
there is always only one version of a policy with a given id, correct? Otherwise it's possible to miss updates here...